doc/source/contributing.rst

```diff
@@ -25,23 +25,24 @@ Or maybe through using *pandas* you have an idea of you own or are looking for s
 in the documentation and thinking 'this can be improved'...you can do something
 about it!
 
-Feel free to ask questions on `mailing list
-<https://groups.google.com/forum/?fromgroups#!forum/pydata>`_
+Feel free to ask questions on the `mailing list
+<https://groups.google.com/forum/?fromgroups#!forum/pydata>`_ or on `Gitter
```

Comment:
Well, I think the 1 trailing underscore is OK in practice (it's also what sphinx says to use: http://sphinx-doc.org/rest.html#external-links), although in theory, if you don't want to reference this link again (eg with `` `mailing list`_ ``), you should use two trailing underscores (http://stackoverflow.com/questions/27420317/restructured-text-rst-http-links-underscore-vs-use)